pkg/cmd/version/version.go

```diff
@@ -0,0 +1,28 @@
+package version
+
+import (
+	"fmt"
+
+	"github.com/coreeng/corectl/pkg/cmdutil/config"
+	"github.com/spf13/cobra"
+)
+
+var (
+	version = "dev"
```

Comment:
Can we also set this values with `make build`/`make install`?

Reply:
![image](https://github.com/user-attachments/assets/48bbc0d2-17d2-4f14-910d-a0e5ff5c936d)
I've added some config to the makefiles so that this works